### PR TITLE
Port GetMailboxesSimulation to RFC-8621

### LINKED ISSUE #84: Port GetMailboxesSimulation to RFC-8621
# Why?

Have our first JMAP RFC-8621 simulation and start transfering method steps to RFC-8621.

# How?

Define the following steps in a jmap-rfc-8621 package :

 - Use basic authentication
 - Don't forget the accept header...
 - Implement Session step to retrieve accountId
 - Perform a `Mailbox/get` call with validations

Copying draft work, implement a scenario and a simulation.

# Definition of done

Have a quick, with low load :green_apple: run of gating on this new simulation.

### PR DESCRIPTION
**Pull request by Arsnael**

### CODE REVIEW

**Review by chibenwa — APPROVED** [review]

**Comment by chibenwa** [comment]
Update the readme accordingly (adding `.draft` in the FQDN, mention the new simulation if needed)

**Review by chibenwa — APPROVED** [review]

File: README.adoc [hunk 1]
```diff
@@ -108,7 +110,22 @@ Authenticate 100 JMAP users, one time. Then they all retrieve their system mailb
...
-=== SmtpNoAuthenticationNoEncryptionMixedBodySimulation
+=== JMAP RFC-8621
+
+==== MailboxGetSimulation
+
+FQDN :
+
+.scala
+----
+org.apache.james.gatling.simulation.jmap.draft.JmapMessageUpdateSimulation
```
Comment (chibenwa) [inline README.adoc]: Wrong FQDN for being in MailboxGetSimulation section I bet

Comment (Arsnael) [reply README.adoc]: Too quick of a bad copy/paste... Thanks!

**Review by chibenwa — APPROVED** [review]

**Comment by Arsnael** [comment]
```
Request 'getMailQueues:spool' failed for user 1: status.find.is(200), but actually found 404
[...]
[error] Failed tests:
[error]         org.apache.james.gatling.jmap.draft.JmapQueueBrowseIT
```
From what I could see when debugging, it's because we change the image of James to `branch-master`. it seems before when starting James we had some queues created, like the one we are trying to check here 'spool', but not anymore. What changed for memory?

File: src/it/scala-2.12/org/apache/james/gatling/jmap/draft/JmapQueueBrowseIT.scala [hunk 2]
```diff
@@ -1,20 +1,26 @@
-package org.apache.james.gatling.jmap.draft
+/*
+ISSUE-84: 404 on trying to fetch the mail queue 'spool' with the upgraded James memory image to branch-master.
+It seems there is potentially a bug where we loose the mail queues on memory version lately.
+Please fix it on James before re-enabling this test.
```
Comment (chibenwa) [inline src/it/scala-2.12/org/apache/james/gatling/jmap/draft/JmapQueueBrowseIT.scala:4]: Reference the link to the james issue please

File: src/main/scala-2.12/org/apache/james/gatling/jmap/rfc8621/JmapMailbox.scala [hunk 3]
```diff
@@ -1,5 +1,28 @@
...
+       |    "urn:ietf:params:jmap:mail"
+       |  ],
+       |  "methodCalls": [[
+       |    "Mailbox/get",
+       |    {
+       |      "accountId": "$${accountId}",
+       |      "ids": null
+       |    },
+       |    "c1"]]
+       |}""".stripMargin)))
```
Comment (LanKhuat) [inline src/main/scala-2.12/org/apache/james/gatling/jmap/rfc8621/JmapMailbox.scala]: indent

**Review by LanKhuat — APPROVED** [review]

**Pull request merged**